zebra-chain/src/sprout/note/nullifiers.rs

```diff
@@ -13,17 +14,17 @@ use super::super::keys::SpendingKey;
 /// https://zips.z.cash/protocol/protocol.pdf#commitmentsandnullifiers
 fn prf_nf(a_sk: [u8; 32], rho: [u8; 32]) -> [u8; 32] {
     let mut state = [0u32; 8];
-    let mut block = [0u8; 64];
+    let mut block: GenericArray<u8, U64> = GenericArray::default();
```

Comment:
I kept the type signature explict but rust compiler is able to infer it. I will remove it if it's unnecessary.

Reply:
lets fixup this one too to be consistent with the other one then I think this is good to go
```suggestion
    let mut block = GenericArray::<u8, U64>::default();
```